drivers/qwiic/micro_oled.c

```diff
@@ -99,12 +102,17 @@ static uint8_t micro_oled_screen_buffer[LCDWIDTH * LCDWIDTH / 8] = {0x00, 0x00,
 #    endif
 #else
 // catchall for custom screen szies
```

Comment:
```suggestion
// catchall for custom screen sizes
```

May as well fix this small typo here :D